shortcodes/epfl-allowed-iframe/epfl-allowed-iframe.php

```diff
@@ -25,7 +25,7 @@ function allowed_url_checker( $url, $allowed_url ) {
     }
 
     // path has to be the same
-    if ( $allowed_url['path'] !== $url['path'] ) {
+     if ( substr($allowed_url['path'], 0, strlen($allowed_url['path'])) !== substr($url['path'], 0, strlen($allowed_url['path'])) ) {
```

Comment:
`substr($allowed_url['path'], 0, strlen($allowed_url['path'])` should be just `$allowed_url['path']`. Please use `strncmp()` as per https://stackoverflow.com/a/2792233 for clarity.